members/urls.py

```diff
@@ -1,7 +1,9 @@
 from django.urls import path
 
-from . import views
+from members.views import index, new_inscription, new_inscription_infos
 
 urlpatterns = [
-    path('', views.index, name='index'),
-]
\ No newline at end of file
+    path('', index, name='index'),
+    path('inscriptions/new/', new_inscription, name='new'),
+    path('inscriptions/new/infos/', new_inscription_infos, name='infos'),
```

Comment:
Please keep relative import, as in exemples https://docs.djangoproject.com/fr/3.1/topics/http/urls/